conda/plugins/subcommands/doctor/health_checks.py

```diff
@@ -163,8 +162,29 @@ def env_txt_check(prefix: str, verbose: bool) -> None:
     print(f"Environment listed in environments.txt file: {present}\n")
 
 
+def requests_ca_bundle_check(prefix: str, verbose: bool) -> None:
+    if not os.getenv("REQUESTS_CA_BUNDLE"):
+        return
+    elif not Path(os.getenv("REQUESTS_CA_BUNDLE")).exists():
+        print(
+            f"{X_MARK} Env var `REQUESTS_CA_BUNDLE` is pointing to a non existent file.\n"
+        )
+    else:
+        try:
+            response = requests.get("https://example.com")
```

Comment:
There are two things I'd like to mention for this line.

In conda, we do not use `requests` directly. Instead, we use the `get_session` function which returns a `CondaSession` object. We should also do this here so the doctor check is as similar to regular conda HTTP requests as possible.

For an example, of how this is used please see:

- https://github.com/conda/conda/blob/6148dda2527f610c658dab8e6172b07b6c64004a/conda/notices/fetch.py#L64

The second thing I'd like to mention is that we should probably use a different URL. I would suggest either `https://conda.org` or `https://anaconda.org`.

@kenodegard, do you have any thoughts on the test URL?

Reply:
Thanks. I will look into the `CondaSession` suggestion. 
So I had the same question. I wanted to use conda.org too. But I learned that https://example.com is a valid website and it is, IIUC, guaranteed to always be available.

Reply:
Both conda.org and anaconda.org will give false negatives for those with restrictive intranets. What if we just use one of the urls the user is expecting to work? e.g.:

```python
conda.models.channel.Channel(conda.base.context.context.channels[0]).urls()[0]
```

Reply:
could also iterate over all of those URLs and report whether or not they work? would be another nice health check for those trying to debug proxy/network issues etc.

Reply:
@kenodegard,

That was my first thought. I vote for that approach.